mantidproject/mantid
# Indirect IDR facility based tab display logic

Fixes [#11158](http://trac.mantidproject.org/mantid/ticket/11158).

To test:
- Ensure current facility is ISIS
- Open Indirect Data Reduction, should have things like spectra range populated for current instrument
- Run something (e.g. Calibration with IRIS 26173)
- Switch facility to SNS
- ISIS tabs will no longer appear, SNS instruments will be in selector, an error will be given about not being able to load the instrument (this is normal)
- Run something else (e.g. SQw moments with `irs53664_graphite002_sqw.nxs`)
- Switch back to ISIS, ISIS tabs should be back
- Run an ISIS tab (e.g. Energy Transfer with 26173, the calibration file created earlier and plot as a contour)

(as a bonus this also tests the calibration and ET routines as you notice the contour plot should be almost perfectly straight)
